File: 05-mayoria-de-edad/python/mayordeedad.py
from datetime import datetime

fecha_nacimiento = input("Introduzca su fecha de nacimiento (dd/mm/aaaa): ")
f_nac = datetime.strptime(fecha_nacimiento, "%d/%m/%Y")
f_actual = datetime.now()
delta = f_actual - f_nac
edad = delta.days // 365
if edad >= 18:
    print("Usted es mayor de edad")
else:
    print("Usted es menor de edad")

File: 05-mayoria-de-edad/php/mayordeedad.php
<?php

$fecha_nacimiento = readline("Introduzca su fecha de nacimiento (dd/mm/aaaa): ");
$f_nac = date_create_from_format("d/m/Y", $fecha_nacimiento);
$fecha_actual = new DateTime();
$edad = $fecha_actual->diff($f_nac);
if ($edad->y >= 18) {
    echo "Eres mayor de edad\n";
} else {
    echo "Eres menor de edad\n";
}